addons/headless/XEH_postInit.sqf

```diff
@@ -1,12 +1,12 @@
 #include "script_component.hpp"
 
-["ace_settingsInitialized", {
+["CBA_settingsInitialized", {
     // Register and remove HCs if not client that is not server and distribution or end mission enabled
     if ((!hasInterface || isServer) && {XGVAR(enabled) || XGVAR(endMission) != 0}) then {
         if (isServer) then {
             // Request rebalance on any unit spawn (only if distribution enabled)
             if (XGVAR(enabled)) then {
-                ["AllVehicles", "initPost", FUNC(handleSpawn), nil, nil, true] call CBA_fnc_addClassEventHandler;
+                ["CAManBase", "initPost", LINKFUNC(handleSpawn), nil, nil, true] call CBA_fnc_addClassEventHandler;
```

Comment:
Does anyone know why `AllVehicles` was chosen over `CAManBase`? In `FUNC(handleSpawn)` we only handle objects in `allUnits`, which can only be objects inheriting from `CAManBase` or am I wrong?

Reply:
I am not sure, it may be a remnant from originally the process working on units and not groups (before `setGroupOwner` existed), but it might also be there to catch something that `CAManBase` doesn't.

Reply:
I was thinking of moving to https://community.bistudio.com/wiki/Arma_3:_Mission_Event_Handlers#GroupCreated, but
> Triggered when a [Group](https://community.bistudio.com/wiki/Group) is created. Note that the group contains no units at that point!
meaning you'd have to add some more code to figured out when the group has units added. Not worth the hassle imo.

---

I wanted to reduce the amount of needless EH being added to vehicles such as cars and tanks. Unfortunately I don't know and can't guarantee if `CAManBase` covers everything, but given it's used in every other component, would it not be adequate?

Reply:
Huh, GitHub did something weird and is showing me same comment here and in review below, even though it's an answer... I deleted it by mistake. It wasn't very useful anyways.

---

`GroupCreated` won't work, no, and it also wouldn't caught units being spawned in with an existing group. Although those should be automatically moved to the same machine as the group leader. I don't feel like trusting that.

Let's switch to `CAManBase`, I have not found a good reason for it to be `AllVehicles`. Entirely possible it's just so old I didn't really know what I was doing (this was one of the first components I wrote for ACE3/ACEX).